scalding-core/src/test/scala/com/twitter/scalding/typed/NoStackLineNumberTest.scala

```diff
@@ -1,51 +0,0 @@
-/*
-Copyright 2015 Twitter, Inc.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-package com.twitter.scalding.typed
-
-import org.scalatest.WordSpec
-
-import com.twitter.scalding._
-import scala.concurrent.{ ExecutionContext => SExecutionContext, _ }
-import SExecutionContext.Implicits.global
-import scala.concurrent.duration.{ Duration => SDuration }
-
-import cascading.flow.FlowDef
-import org.apache.hadoop.conf.Configuration
-
-class NoStackLineNumberTest extends WordSpec {
-
-  "No Stack Shouldn't block getting line number info" should {
```

Comment:
can we replace this with an equivalent test that we can see the caller correctly?